cycledash/static/scss/examine.scss

```diff
@@ -12,39 +12,22 @@ a, a:hover {
   color: #000;
   text-decoration: none;
 }
-body {
-  font-size: 16px;
-  line-height: 1;
-  margin: 8px;
-}
 
-/* VCFTable */
+/*------[ VCF Table ]------*/
 .vcf-table {
-  padding-top: 20px;
+  @extend .table, .table-striped, .table-condensed;
```

Comment:
Is there a reason to `@extend` here rather than just add these classes to the `.vcf-table`? 

As an aside: when you `@extend` something, does that mean the CSS from that class is pulled into this file?